contracts/CandidateHub.template

```diff
@@ -154,11 +155,10 @@ contract CandidateHub is ICandidateHub, System, IParamSubscriber {
           b. transfer the candidate's margin eth value to the SystemReward contract
   */    
   function jailValidator(address operateAddress, uint256 round, uint256 fine) 
-        external override onlyValidator onlyIfCandidateAddress {
-    uint256 index = operateMap[operateAddress];
-    if (index == 0) return;
-
-    Candidate storage c = candidateSet[index - 1];
+        external override onlyValidator onlyIfCandidateAddress(operateAddress) {
+    uint256 indexPlus1 = operateMap[operateAddress];
+    uint index_ = indexPlus1 - 1;
+    Candidate storage c = candidateSet[index_];
     uint256 margin = c.margin;
     if (margin >= dues && margin - dues >= fine) {
```

Comment:
@jackcrypto9527 SafeMath has been added directly to the compiler since 0.8. it's there even if you do not explicitly call it.  Anyway code has been reverted